Kudu.Contracts/Diagnostics/ProcessInfo.cs

```diff
@@ -30,6 +30,12 @@ public class ProcessInfo : INamedObject
         [JsonProperty(PropertyName = "is_profile_running", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public bool IsProfileRunning { get; set; }
 
+        [JsonProperty(PropertyName = "is_iis_profile_running", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public bool IsIisProfileRunning { get; set; }
+
+        [JsonProperty(PropertyName = "iis_profile_timeout_seconds", DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
I would say let's call this: iis_profile_timeout_in_seconds to be compat with rest of the code